### PR TITLE
Add current_sign_in_at to api User

### PR DESCRIPTION
**Pull request by sselberg**
Allows admins to track user activity over time.

http://feedback.gitlab.com/forums/176466-general/suggestions/7481401-track-user-activity-through-rest-api

### CODE REVIEW

File: doc/api/users.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ GET /users
     "is_admin": false,
     "avatar_url": "http://localhost:3000/uploads/user/avatar/1/cd8.jpeg",
     "can_create_group": true
+    "current_sign_in_at" : "2014-03-19T13:12:15Z"
```
Comment (jvanbaarsen) [inline doc/api/users.md]: Can you remove the space between `"current_sign_in_at" :`?

File: doc/api/users.md [hunk 2]
```diff
@@ -80,6 +81,7 @@ GET /users
     "can_create_group": true,
     "can_create_project": true,
     "projects_limit": 100
+    "current_sign_in_at" : "2014-03-19T17:54:13Z"
```
Comment (jvanbaarsen) [inline doc/api/users.md]: Same as above

**Comment by jvanbaarsen** [comment]
@sselberg Can you please fix the space and rebase your commits so its one commit? Ping me when ready :)

**Comment by sselberg** [comment]
@jvanbaarsen 
Done!
Sorry for the delay.

**Comment by jvanbaarsen** [comment]
@sselberg Thanks

File: CHANGELOG [hunk 3]
```diff
@@ -138,6 +138,7 @@ v 7.9.3
 
 v 7.9.3
   - Contains no changes
+  - Add current_sign_in_at to UserFull REST api.
```
Comment (randx) [inline CHANGELOG]: @sselberg Please move CHANGELOG to 7.11

**Comment by sselberg** [comment]
@randx 
Moved CHANGELOG entry to v7.11.0.

**Comment by sselberg** [comment]
I don't get how the build fail is related to this change.
I find no diff in any of the files changed between the commit that passed and this one.

**Comment by sselberg** [comment]
@randx 
Looks like the build failed after I rebased on top of origin/master...
...master broken?

**Comment by Razer6** [comment]
@sselberg master was broken but it's already fixed. Can you rebase to get this fix - then your build will also be green :)

**Comment by sselberg** [comment]
@Razer6 
Yep, seems to work now, so far at least.
Thanks!

**Comment by sselberg** [comment]
@randx 
Moved CHANGELOG entry to v7.11.0.

File: doc/api/users.md [hunk 4]
```diff
@@ -58,6 +58,7 @@ GET /users
     "is_admin": false,
     "avatar_url": "http://localhost:3000/uploads/user/avatar/1/cd8.jpeg",
     "can_create_group": true
```
Comment (SidRoberts) [inline doc/api/users.md]: There's a comma missing at the end of the line. Note the red colon on the next line.

File: doc/api/users.md [hunk 5]
```diff
@@ -80,6 +81,7 @@ GET /users
     "can_create_group": true,
     "can_create_project": true,
     "projects_limit": 100
```
Comment (SidRoberts) [inline doc/api/users.md]: See above (line 60).

**Comment by sselberg** [comment]
@SidRoberts 
"There's a comma missing at the end of the line. Note the red colon on the next line."
Done!

**Comment by sselberg** [comment]
Once again the build failed on something not related to my change [1] , but this time the parent commit is the same.
Seems like a shaky test.
Is there any way that I can retrigger a job in Semaphore?

[1]
"Error summary:
  Errors (1)
    Project Merge Requests :: I show comments on a merge request diff with comments in multiple files :: Then I should see a comment like "Line is wrong" in the second file"

**Comment by sselberg** [comment]
A ' $ git commit --amend' fixed the build.
So yes, it's shaky :-)

**Comment by randx** [comment]
Thanks

**Pull request merged**